docs/1.3_to_1.4_migration_guide.md

```diff
@@ -0,0 +1,15 @@
+# List of broken changes for migration from indy-node 1.3 to 1.4:
+
```

Comment:
Please mention a migration script that will be called during upgrade. 
Please mention that it will be called automatically if a Node is updated via POOL_UPGRADE txn, and that it needs to be called manually if a node was updated manually.